src/userdoc/adding-features/nuxeo-imap-connector.md

```diff
@@ -206,11 +206,24 @@ The emails imported in the email folder cannot be modified in the Nuxeo Platform
 
 **To create an email folder**
 
+Required below changes in Gmail account.
+
+1. Turn on [2-step verification](https://support.google.com/accounts/answer/185839?hl=en&co=GENIE.Platform%3DAndroid).
```

Comment:
Can you cleanup and update the url to `https://support.google.com/accounts/answer/185839` instead? (no need to keep the `?hl=en&co=GENIE.Platform%3DAndroid` part 🙂)

Reply:
resolved this issue